lms/djangoapps/branding/views.py

```diff
@@ -102,3 +103,7 @@ def courses(request):
     #  we do not expect this case to be reached in cases where
     #  marketing is enabled or the courses are not browsable
     return courseware.views.courses(request)
+
+
+def footer(request):
```

Comment:
Also, please cache the response here.  Django has a built-in decorator that will do this:
https://docs.djangoproject.com/en/1.4/topics/cache/#the-per-view-cache